dope_flag: true
err_file: rayTracing
gazebo_flag: false
object_name_list:
- cracker
- soup
- jelloR
object_num: 2
other_obj_num: 1
oto_name_list:
- base_of_cracker
- fish_can
otob_name_list:
- cracker_obstacle
- fish_can
particle_num: 30
pick_particle_rate: 0.00
robot_num: 1
run_alg_flag: PBPF
task_flag: '3'
update_style_flag: time
version: multiray
optitrack_flag: False
obstacles_pos: [[0.8315, 0.215, 0.14765408574692843]]
obstacles_ori: [[1.0, 0.0, 0.0, 1.0]]

